app/src/renderer/components/wallet/SendModal.vue

```diff
@@ -174,25 +119,15 @@ export default {
     ToolBar,
     ActionModal
   },
-  props: {
-    denom: {
-      type: String,
-      required: true
-    }
-  },
   data: () => ({
     bech32error: null,
-    fields: {
-      address: ``,
-      amount: null,
-      denom: ``,
-      password: ``
-    },
-    sending: false
+    address: ``,
+    amount: ``,
```

Comment:
some of the modals on this PR use `""` and some use `null`, imo we should stick with `null` bc `""`  is a different type and looks weird